cmd/machine-config-daemon/pivot.go

```diff
@@ -193,25 +194,29 @@ func podmanRemove(cid string) {
 }
 
 // getDefaultDeployment uses rpm-ostree status --json to get the current deployment
-func getDefaultDeployment() types.RpmOstreeDeployment {
+func getDefaultDeployment() (types.RpmOstreeDeployment, error) {
 	// use --status for now, we can switch to D-Bus if we need more info
 	var rosState types.RpmOstreeState
+	var errRet types.RpmOstreeDeployment
```

Comment:
oh, usually you return `types.RpmOstreeDeployment{}` in the return statements in go. Actually, this might as well be a pointer so we can return `nil, err`